src/Grid/Grid.php

```diff
@@ -1336,6 +1341,10 @@ public function getTweaks()
         return $this->tweaks;
     }
 
+    public function getAllTweaks(): ?array {
```

Comment:
Would it make more sense for this to always be an array, rather than could be null? So what's the difference between null and an empty array?

Reply:
Updated